packages/integration.rb

```diff
@@ -0,0 +1,74 @@
+require 'package'
+
+class Integration < Package
+  description 'Improves integration with Chrome OS shell and Chromebrew GUI applications.'
+  homepage 'https://github.com/skycocker/chromebrew/crew_integration'
+  version '3.0'
+  source_url 'file:///dev/null'
+  source_sha256 'e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855'
+  
+  depends_on "graphicsmagick"
+    
+  case ARGV[0]
+  when 'reinstall'
+     puts "Since you are reinstalling this package, please run 'pkill ruby' if installation failed".lightblue
+  when 'upgrade'
+     puts "Since you are upgrading this package, please run 'pkill ruby' if installation failed".lightblue
+  end
+
+  def self.install
+    # system "curl https://codeload.github.com/skycocker/chromebrew/zip/master -#o crew_integration.zip"
+    system "curl https://codeload.github.com/supechicken/chromebrew/zip/supechicken-patch-1 -#o crew_integration.zip"
+    puts 'Archive downloaded'.lightgreen
+    system "unzip crew_integration.zip > /dev/null"
```

Comment:
No need for `> /dev/null`.  It's fine (and better for debugging, if necessary) to show the output.

Reply:
Removed